cluster-scope/base/apps/daemonsets/kvm-device-plugin/daemonset.yaml

```diff
@@ -30,6 +30,9 @@ spec:
       containers:
       - name: kvm-device-plugin
         image: kvm-device-plugin:latest
+        resources:
+          requests:
+            cpu: 100m
```

Comment:
what about `requests.memory`, `limits.cpu`, `limits.memory` ?

Reply:
Memory doesn't seem to be problematic, so I'd go with the defaults.
I'd also not want to artificially limit the CPU in case more is needed and available.

Reply:
Ok the quota looks at the limits, not the requested amount. With limit 100m and request 20m the containers successfully start. Still have to see how they behave with a CoreOS Assembler build running..

Reply:
I was wrong. We'll also need to limit memory. Added.